README.md

```diff
@@ -1 +1,58 @@
-# terraform-alicloud-fc
\ No newline at end of file
+Alicloud Function Compute (FC) Terraform Module
+=============================================
+
+Terraform module which creates FC resources on Alibaba Cloud.
+
+These types of resources are supported:
+
+* [service](https://www.terraform.io/docs/providers/alicloud/r/fc_service.html)
+* [function](https://www.terraform.io/docs/providers/alicloud/r/fc_function.html)
+* [trigger](https://www.terraform.io/docs/providers/alicloud/r/fc_trigger.html)
+
+Root module calls these modules which can also be used separately to create independent resources:
+
+* [service](https://github.com/terraform-alicloud-modules/terraform-alicloud-fc/tree/master/modules/service) - creates Service
+* [function](https://github.com/terraform-alicloud-modules/terraform-alicloud-fc/tree/master/modules/function) - creates Function
+* [trigger](https://github.com/terraform-alicloud-modules/terraform-alicloud-fc/tree/master/modules/trigger) - creates Trigger
+
+Usage
+-----
+You can use this in your terraform template with the following steps.
+
+1. Adding a module resource to your template, e.g. main.tf
+
+
+         module "tf-fc" {
+            source = "alibaba/fc/alicloud"
```

Comment:
Line 26 should be `source = "terraform-alicloud-modules/alibaba/fc/alicloud"`

Reply:
done